src/app/components/sessions/sessions.component.html

```diff
@@ -0,0 +1,104 @@
+                          </div>
+                          <input required class="form-control" [(ngModel)]="session.location"  type="text" name="location" id="location">
+                        </div>
+                    </div>
+                    <div class="col-md-5">
+                        <div class="input-group  mb-2">
+                            <div class="input-group-prepend">
+                                <span class="input-group-text">{{'START_TIME' | translate}}</span>
+                            </div>
+                            <input  class="form-control" [(ngModel)]="session.startTime"  required style="width: 160px"  type="time" name="startDate" id="startDate"  >
+                        </div>
+                      <div class="input-group  mb-2">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text">{{'END_TIME' | translate}}</span>
+                        </div>
+                        <input   class="form-control" [(ngModel)]="session.endTime"  required style="width: 160px"   type="time" name="endDate" id="endDate" >
+                      </div>
+                    </div>
+                  </div>
+                  <div class="row">
+                    <div class="col-md-10">
+                      <div class="input-group mb-2">
+                          <div class="input-group-prepend">
+                              <span class="input-group-text">{{'NOTES' | translate}}</span>
+                          </div>
+                          <textarea required class="form-control" [(ngModel)]="session.notes"  type="text" name="notes" id="notes"> </textarea>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="col-md-2">
+                  <div class="row mt-3 mb-2">
+                    <button type="button"  data-toggle="tooltip" data-placement="top" title="{{'SAVE' | translate}}" (click)='addingSession();' class="btn btn-outline-success"><i class="fa fa-floppy-o" aria-hidden="true">  {{'SAVE' | translate}}</i>
+                    </button>
+                  </div>
+                  <div class="row mb-5">
+                    <button type="button"  data-toggle="tooltip" data-placement="top" title="{{'CANCEL' | translate}}" (click)='addSession = false' class="btn btn-outline-danger mr-5"><i class="fa fa-ban" aria-hidden="true"> {{'CANCEL' | translate}}</i>
+                    </button>
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
+</div>
+
+<div *ngIf="searchSession" class="container row" > 
+    <div class="card col-md-3" *ngFor="let session of listSessions | filterSession:filterSession">
+      <app-session-card (sessionDeleted)="refreshSessions();" [session] = "session" [batchId]= "session.batchId"> </app-session-card>
```

Comment:
If you are passing the object session to app-session-card why do you need to pass also session.batchId?